cmd/security-secrets-setup/testdata/test-example-security.yml

```diff
@@ -0,0 +1,625 @@
+# /*******************************************************************************
+#  * Copyright 2019 Intel Corporation
+#  *
+#  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+#  * in compliance with the License. You may obtain a copy of the License at
+#  *
+#  * http://www.apache.org/licenses/LICENSE-2.0
+#  *
+#  * Unless required by applicable law or agreed to in writing, software distributed under the License
+#  * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  * or implied. See the License for the specific language governing permissions and limitations under
+#  * the License.
+#  *
+#  * @author: Jim White, Dell
+#  * added: July 26, 2019
+#  * @author: Jim Wang, Intel Corp
+#  * updated: October 8, 2019
+#  *******************************************************************************/
+
+version: '3.4'
+
+# all common shared environment variables defined here:
+x-common-env-variables: &common-variables
+  edgex_registry: consul://edgex-core-consul:8500
+  Clients_CoreData_Host: edgex-core-data
+  Clients_Logging_Host: edgex-support-logging
+  Logging_EnableRemote: "true"
+
+volumes:
+  db-data:
+  log-data:
+  consul-config:
+  consul-data:
+  consul-scripts:
+  portainer_data:
+  vault-init:
+  vault-config:
+  vault-file:
+  vault-logs:
+  secrets-setup-cache:
+
+services:
+  volume:
+    image: nexus3.edgexfoundry.org:10004/docker-edgex-volume:1.1.0
+    container_name: edgex-files
+    networks:
+      - edgex-network
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+
+  consul:
+    #image: nexus3.edgexfoundry.org:10004/docker-edgex-consul:1.1.0
+    image: edgexfoundry/docker-edgex-consul:1.1.1
+    ports:
+      - "8400:8400"
+      - "8500:8500"
+    container_name: edgex-core-consul
+    hostname: edgex-core-consul
+    networks:
+      edgex-network:
+        aliases:
+            - edgex-core-consul
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+      - consul-scripts:/consul/scripts
+      - vault-config:/vault/config
+      - /run/edgex/secrets:/run/edgex/secrets:ro    
```

Comment:
Make it `/tmp` instead of `/run` by default for the host bindings or the dockerfile won't work on Macs.  Adjust env var overrides to match.

Reply:
Request to only mount `/tmp/edgex/secrets/ca`  --  consul just needs to know that pki init ran, not have access to everyone's private keys.

Reply:
- not sure that is the case that consul just needs ca because of dependency checking scripts
- /tmp will be adjust in another separate PR, not this one, trying to reduce the size of PR and separate changes as much as I can

Reply:
We have to change the consul script anyway, because it is looking in /vault/config for the PKI.

Reply:
yea, the changes to tmp is kind of wide-spread out on several places and repos. so I would prefer to do that in separate PRs not in this one yet.